powerfulseal/cli/pscmd.py

```diff
@@ -53,7 +54,7 @@ def colour_output(output, extras=None):
     else:
         pattern = DEFAULT_COLOR_KEYWORDS
     for word, colour in pattern.items():
-        output = output.replace(word, colored(word, colour))
+        output = output.replace(word+'=', colored(word, colour)+'=')
```

Comment:
Isn't this going to stop coloring UP, DOWN and Running?

Reply:
I guess if going this way, you could also add a `=word` variant.

Reply:
Even if I added the `=word` pattern, UP and DOWN  would still not work... (NodeState.UP).

Maybe we can add NodeState to the UP and DOWN keywords.

Reply:
btw https://github.com/bloomberg/powerfulseal/commit/94343afe5e213f3db860491eccd33c2a70c31f28